pkg/provisioner/ironic/ironic.go

```diff
@@ -1295,3 +1324,23 @@ func (p *ironicProvisioner) PowerOff() (result provisioner.Result, err error) {
 
 	return result, nil
 }
+
+// getImageChecksum parses the checksum; splits and returns image_hash and image_value
```

Comment:
It would be useful to see some example values in the comments here to understand what this function is trying to do when it parses its input. This function could also use some unit tests.

Reply:
> It would be useful to see some example values in the comments here to understand what this function is trying to do when it parses its input. This function could also use some unit tests.

+1. Will add examples and unit tests. Thanks for the feedback.